### PR TITLE
Sample Scenario for  WS-Federation Single Sign-On

### PR DESCRIPTION
**Pull request by asekawa**

### CODE REVIEW

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md [hunk 1]
```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
...
+
+2.  Navigate to **Main**>**Identity**>**Service Providers** and click **Add**.
+
+3.  Enter **PassiveSTSSampleApp** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4.  In the **Inbound Authentication Configuration** section, set following configurations
+    under the **WS-Federation (Passive) Configuration** section.
+
+    1.
```
Comment (nilminiwso2) [inline en/docs/guides/login/configure-ws-federation-single-sign-on.md:138]: should this be removed?

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md [hunk 2]
```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
...
+
+3.  Enter **PassiveSTSSampleApp** in the **Service Provider Name** text box,
+    and click **Register**.
+
+4.  In the **Inbound Authentication Configuration** section, set following configurations
+    under the **WS-Federation (Passive) Configuration** section.
+
+    1.  
+
+        1.  **Passive STS Realm:** : `               PassiveSTSSampleApp             `
```
Comment (nilminiwso2) [inline en/docs/guides/login/configure-ws-federation-single-sign-on.md:140]:
```suggestion
        -  **Passive STS Realm:** : `               PassiveSTSSampleApp             `
```

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md [hunk 3]
```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
...
+
+        2.  **Assertion Consumer URL** : `
```
Comment (nilminiwso2) [inline en/docs/guides/login/configure-ws-federation-single-sign-on.md:142]:
```suggestion
        -   **Assertion Consumer URL** : `
```

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md [hunk 4]
```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
...
+            http://localhost:8080/PassiveSTSSampleApp/index.jsp`
+                 
+            Click Yes, in the message that appears.
+
+    
+
+5. Expand the **Claim Configuration** section and click **Add Requested Claim** to add the following claims. 
+    1. http://wso2.org/claims/username
```
Comment (nilminiwso2) [inline en/docs/guides/login/configure-ws-federation-single-sign-on.md:150]:
```suggestion
    -  http://wso2.org/claims/username
```

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md [hunk 5]
```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
...
+    2. http://wso2.org/claims/emailaddress
```
Comment (nilminiwso2) [inline en/docs/guides/login/configure-ws-federation-single-sign-on.md:151]:
```suggestion
    -  http://wso2.org/claims/emailaddress
```

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md [hunk 6]
```diff
@@ -9,11 +9,176 @@ This page guides you through enabling [single sign-on](../../../references/conce
...
+    
+5. Restart the tomcat server.
+
+To check the sample application,
+navigate to `http://<TOMCAT_HOST>:<TOMCAT_PORT>/PassiveSTSSampleApp/home.jsp ` on your browser.
+
+For example, ` http://localhost:8080/PassiveSTSSampleApp/home.jsp .`
+
+
+### Configuring the service provider
```
Comment (nilminiwso2) [inline en/docs/guides/login/configure-ws-federation-single-sign-on.md:124]:
```suggestion
### Configure the service provider
```

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md (hunk 1 above)
Comment (asekawa) [reply en/docs/guides/login/configure-ws-federation-single-sign-on.md]: commit 13cc337adbc3c694220614bbf71b3e343379056b

File: en/docs/guides/login/configure-ws-federation-single-sign-on.md (hunk 2 above)
Comment (asekawa) [reply en/docs/guides/login/configure-ws-federation-single-sign-on.md]: commit 13cc337adbc3c694220614bbf71b3e343379056b

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**